### PR TITLE
dev-util/pycharm-professional: add 2021.3.2

### PR DESCRIPTION
**Pull request by 5cat**
added new version pycharm-professional with the help of Ionen https://forums.gentoo.org/viewtopic-p-8693272.html

### CODE REVIEW

**Comment by 5cat** [comment]
Hello, it says that I need to amend the GCO to my commit, but the GCO asks for the legal name and I prefer not to include it. Is there a way I can contribute without including my legal name? 

It is my first time contributing so I'm not familiar yet with the process.

**Review by ionenwks — COMMENTED** [review]
lgtm beside two small last minute things I see (note it's fine to --force push to update PR)

Lack of GCO signoff is more of a problem though, it's needed for non-trivial contributions (if very trivial could merge under my name with a Thanks-to, but in this case the ebuild was essentially rewritten). Proper signoff would certainly make things simpler if want to contribute, which I'd personally like to see more of.

File: dev-util/pycharm-professional/pycharm-professional-2021.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+	readme.gentoo_create_doc
+
+	# recommended by: https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit
+	insinto /usr/lib/sysctl.d
+	newins - 30-idea-inotify-watches.conf <<<"fs.inotify.max_user_watches = 524288"
+}
+
+pkg_postinst() {
+	xdg_pkg_postinit
```
[the remark this answers is not in the document]
Comment (5cat) [reply dev-util/pycharm-professional/pycharm-professional-2021.3.2.ebuild]: Thank you Ionen, didnt know about the `/usr/share/pixmaps` part. amended the commit with the xdg removal

File: dev-util/pycharm-professional/pycharm-professional-2021.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	app-accessibility/at-spi2-atk:2
+	app-accessibility/at-spi2-core:2
+	dev-libs/atk
+	dev-libs/expat
+	dev-libs/glib:2
+	dev-libs/nspr
+	dev-libs/nss
+	media-libs/alsa-lib
+	media-libs/freetype
+	media-libs/mesa
```
[the remark this answers is not in the document]
Comment (5cat) [reply dev-util/pycharm-professional/pycharm-professional-2021.3.2.ebuild]: done, amended to the commit

**Comment by 5cat** [comment]
> lgtm beside two small last minute things I see (note it's fine to --force push to update PR)
> 
> Lack of GCO signoff is more of a problem though, it's needed for non-trivial contributions (if very trivial could merge under my name with a Thanks-to, but in this case the ebuild was essentially rewritten). Proper signoff would certainly make things simpler if want to contribute, which I'd personally like to see more of.

While I don't like the idea of giving out too much identifiable information, I understand that things can get complicated if this is not done. And since I like the distro/community and will continue contributing in the future I added the GCO sign off to the commit. will check the gentoo forums later regarding the GCO.

**Review by ionenwks — APPROVED** [review]
> While I don't like the idea of giving out too much identifiable information, I understand that things can get complicated if this is not done. And since I like the distro/community and will continue contributing in the future I added the GCO sign off to the commit. will check the gentoo forums later regarding the GCO.

Yeah it's understandable, there's often debates about the whole thing given it can stop some from wanting to contribute -- but it's the rules we're stuck with for the foreseeable future. Thanks.

That aside, PR looks good, I'll go ahead and merge it. Thanks for working on this :+1:

**Pull request closed without merging**